package.json

```
@@ -25,7 +25,7 @@
   },
   "engine-strict": true,
   "engines": {
-    "node": ">= 16"
+    "node": "18"
```

```suggestion
    "node": ">= 18"
```
We should allow any version of node that is 18 or above instead of just 18

---

Noted. Thanks!